airbyte-integrations/connectors/destination-redis/src/test-integration/java/io/airbyte/integrations/destination/redis/RedisHCacheTest.java

```diff
@@ -28,7 +28,7 @@ void init() {
     var jsonConfig = RedisDataFactory.jsonConfig(
         redisContainer.getHost(),
         redisContainer.getFirstMappedPort());
-    redisCache = new RedisHCache(new RedisConfig(jsonConfig));
+    redisCache = new RedisHCache(jsonConfig);
```

Comment:
minor nit: this is outside the scope of this work but since you're already making this change, could we keep the json config naming the same across the tests to use either `jsonConfig` or `configJson`? Preference would be to have `jsonConfig` since that matches the `RedisDataFactory#jsonConfig` method signature

Reply:
May sense